src/providers/MeetingProvider/MeetingManager.ts

```diff
@@ -317,18 +322,15 @@ export class MeetingManager implements AudioVideoObserver {
       configuration,
       logger,
       deviceController,
-      this.eventReporter
+      this.eventController
     );
+    if (!this._eventController) {
+      this._eventController = this.meetingSession.eventController;
+      this._eventController.addObserver(this.eventDidReceiveRef);
```

Comment:
This looks duplicated and we already add an observer before calling initializeMeetingSession. Can you pull the removal and addition logic after the `MeetingSession` is created since, before that the default `EventController` may have not been created right internally.

Reply:
This is to grab the DefaultMeetingSession's EventController if none is provided, it has to be done here as this is currently where we create the MeetingSession

Reply:
Yup I meant remove this one: https://github.com/aws/amazon-chime-sdk-component-library-react/pull/753/files#diff-6e7e64f1faf69cc598827e0ff6e12ed9d76d6984f353ebda6d726571ec39ae4fR242

Reply:
Updated